test/test_helper.exs

```diff
@@ -1,3 +1,3 @@
 Mix.shell(Mix.Shell.Process)
-File.rm_rf("test/tmp")
+
```

Comment:
I don't think this should be removed. The old results need to be cleared out

Reply:
Are you sure? That was in the commit that we don't want.

Reply:
Oh, that commit wasn't clean. There's another change for the tmp directory path in it that looks suspicious. `Path.expand("../../tmp", __DIR__)` was changed to `Path.expand("test/tmp")`, but that got reverted too. I assume that goes back as well. The `rm_rf` seems like it should match paths. I guess this is test code, so it's slightly less important if you've already made a release.